src/ripple/protocol/impl/STLedgerEntry.cpp

```diff
@@ -78,7 +78,7 @@ void STLedgerEntry::setSLEType ()
     type_ = mFormat->getType ();
     if (!setType (mFormat->elements))
     {
-        if (auto j = debugLog())
+        if (auto j = debugLog().error())
```

Comment:
Good question.  As it stands at the moment a Journal is really just a wrapper around a reference to a `Sink`: https://github.com/ripple/rippled/blob/develop/src/ripple/beast/utility/Journal.h#L69.  What really matters is whether the `Sink` goes away.  And, at least for now, there's other machinery managing the lifetime of `Sink`s.